questionpy/_attempt.py

```diff
@@ -243,6 +265,33 @@ def jinja2(self) -> jinja2.Environment:
     def variant(self) -> int:
         return self.attempt_state.variant
 
+    def call_js(
+        self,
+        module: str,
+        function: str,
+        data: JsonValue = None,
+        if_role_feedback: JsModuleCallRoleFeedback | None = None,
+    ) -> None:
+        """Call a javascript function when the LMS displays this question attempt.
+
+        Args:
+            module: JS module name specified as:
+                @[package namespace]/[package short name]/[subdir]/[module name] (full reference) or
+                TODO [subdir]/[module name] (referencing a module within the package where this class is subclassed) or
+                TODO attempt/[module name] (referencing a dynamically created module returned as an attempt file)
```

Comment:
Können wir irgendwie mögliche Kollisionen mit einem [subdir] namens "attempt" vermeiden?

Reply:
Eventuell ist auch die Frage, ob wir diesen Fall (attempt files) wirklich unterstützen wollen/müssen. Es gibt ja außerdem noch andere Bereiche, in denen das Paket Dateien ablegen kann.
Für diese Nicht-Standard-Fälle könnten wir auch QPy-URIs vorschreiben. Initial würde ich das aber eh noch nicht implementieren und können wir später noch genauer überlegen.